pkg/seeds/seeds.go

```diff
@@ -173,8 +167,51 @@ func SeedRpms(db *gorm.DB, repo *models.Repository, size int) error {
 	if err := db.Table(models.TableNameRpmsRepositories).Create(&repositories_rpms).Error; err != nil {
 		return err
 	}
-
 	return nil
+
+}
+
+// createOrgId aims to mainly create most entities in the same org
+// but also create some in other random orgs
+func createOrgId(existingOrgId string) string {
+	orgId := "4234"
+	if existingOrgId != "" {
+		orgId = existingOrgId
+	} else {
+		randomNum := rand.Intn(5)
+		if randomNum == 3 {
+			orgId = strconv.Itoa(rand.Intn(99))
+		}
+	}
+	return orgId
+}
+
+func createVersionArray(existingVersionArray *[]string) []string {
+	if existingVersionArray != nil {
+		return *existingVersionArray
+	}
+	var versionArray []string
+	for k := rand.Intn(4); k < len(config.DistributionVersions); k++ {
+		ver := config.DistributionVersions[k].Label
+		versionArray = append(versionArray, ver)
+	}
+	return versionArray
```

Comment:
It isn't exactly an error, but if `k=3` then `versionArray` is `null`. I feel like we generally expect versions to be `[]` if it's empty because `FillDefaults` makes it `[]` on `createRepository` in the handler. Seeding with null here would cause inconsistencies with the seeded data vs the API data. Do we care about that?

Reply:
good catch, fixed!